src/v2/models/AppState.js

```diff
@@ -53,8 +53,9 @@ export default Model.extend({
     showSignoutLink: {
       deps: ['currentState'],
       fn: function (currentState = {}) {
-        // hide signout for IDENTIFY step
-        return _.isFunction(currentState.cancel) && currentState.step !== 'IDENTIFY';
+        const invalidSignOutSteps = ['IDENTIFY', 'ENROLL', 'SUCCESS'];
+        // hide signout for IDENTIFY and ENROLL step
```

Comment:
- nit: `hide signout for IDENTIFY, ENROLL and SUCCESS step
- do you know why even `success` has `cancel` link?

Reply:
Yeah, the API will send `cancel` in the SUCCESS state. According to @iancorcoran-okta you could still cancel without redeeming for tokens. We have asked Tom to confirm this.